apps/extract-stack/src/extract-merge-request-commits.ts

```diff
@@ -4,23 +4,52 @@ import { Clerk } from "@clerk/clerk-sdk-node";
 import { createClient } from "@libsql/client";
 import { drizzle } from "drizzle-orm/libsql";
 import { Config } from "sst/node/config";
-import { mergeRequestCommits, namespaces, repositories, mergeRequests } from "@acme/extract-schema";
+import { mergeRequestCommits, namespaces, repositories, mergeRequests, RepositorySchema, NamespaceSchema, MergeRequestSchema } from "@acme/extract-schema";
 import { EventHandler } from "sst/node/event-bus";
 import { extractMergeRequestsEvent } from "./events";
-import { QueueHandler } from "./create-message";
-import { extractMergeRequestCommitMessage, type extractMergeRequestData } from "./messages";
+import { createMessageHandler } from "./create-message";
+import { MessageKind, metadataSchema } from "./messages";
+import { z } from "zod";
 
-const clerkClient = Clerk({ secretKey: Config.CLERK_SECRET_KEY });
-const client = createClient({
-  url: Config.DATABASE_URL,
-  authToken: Config.DATABASE_AUTH_TOKEN,
+export const mrcsh = createMessageHandler({
```

Comment:
MergeRequestCommitMessageHandler is too long and the stack fails to build for some reason.